tests/integration_tests/test_types.py

```diff
@@ -0,0 +1,329 @@
+    geth_rpc = geth.w3.provider
+    make_same_rpc_calls(eth_rpc, geth_rpc, "eth_getCode", ["0x57f96e6b86cdefdb3d412547816a82e3e0ebf9d2", "latest"])
+
+    # Do an ethereum transfer
+    contract = deploy_and_wait(ethermint.w3)
+    code = eth_rpc.make_request(
+        'eth_getCode', [contract.address, "latest"]
+    )
+    expected = {
+    "id": "4",
+    "jsonrpc": "2.0",
+    "result": "0x"
+    }    
+    res, err = same_types(code, expected)
+    assert res, err
+
+
+def test_getBlockTransactionCountByNumber(ethermint, geth):
+    eth_rpc = ethermint.w3.provider
+    geth_rpc = geth.w3.provider
+    make_same_rpc_calls(eth_rpc, geth_rpc, "eth_getBlockTransactionCountByNumber", ["0x0"])
+
+    make_same_rpc_calls(eth_rpc, geth_rpc, "eth_getBlockTransactionCountByNumber", ["0x100"])
+
+    tx_hash = send_and_get_hash(ethermint.w3)
+
+    tx_res = eth_rpc.make_request('eth_getTransactionByHash', [tx_hash])
+    block_hash = tx_res['result']['blockNumber']
+    block_res = eth_rpc.make_request('eth_getBlockTransactionCountByNumber', [block_hash])
+    
+
+    expected = {    
+    "id": "1",
+    "jsonrpc": "2.0",
+    "result": "0x0"
+    }
+    res, err = same_types(block_res, expected)
+    assert res, err
+
+
+# TODO: BYHASH
+# def test_getBlockTransactionCountByHash
+
+
+def test_getTransaction(ethermint, geth):
+    eth_rpc = ethermint.w3.provider
+    geth_rpc = geth.w3.provider
+    make_same_rpc_calls(eth_rpc, geth_rpc, "eth_getTransactionByHash", ["0x5c504ed432cb51138bcf09aa5e8a410dd4a1e204ef84bfed1be16dfba1b22060"])
+
+    tx_hash = send_and_get_hash(ethermint.w3)
+
+    tx_res = eth_rpc.make_request('eth_getTransactionByHash', [tx_hash])
+
+    expected = {
+    "jsonrpc": "2.0",
+    "id": 0,
+    "result": {
+        "hash": "0x88df016429689c079f3b2f6ad39fa052532c56795b733da78a91ebe6a713944b",
+        "blockHash": "0x1d59ff54b1eb26b013ce3cb5fc9dab3705b415a67127a003c3e61eb445bb8df2",
+        "blockNumber": "0x5daf3b",
+        "from": "0xa7d9ddbe1f17865597fbd27ec712455208b6b76d",
+        "gas": "0xc350",
+        "gasPrice": "0x4a817c800",
+        "input": "0x68656c6c6f21",
+        "nonce": "0x15",
+        "r": "0x1b5e176d927f8e9ab405058b2d2457392da3e20f328b16ddabcebc33eaac5fea",
+        "s": "0x4ba69724e8f69de52f0125ad8b3c5c2cef33019bac3249e2c0a2192766d1721c",
+        "to": "0xf02c1c8e6114b1dbe8937a39260b5b0a374432bb",
+        "transactionIndex": "0x41",
+        "v": "0x25",
+        "value": "0xf3dbb76162000"
+    }
+    }
+    res, err = same_types(tx_res, expected)
+    assert res, err
+
+# NOT IMPLEMENTED
+# def test_getTransactionRaw(ethermint, geth):
+#     eth_rpc = ethermint.w3.provider
+#     geth_rpc = geth.w3.provider
+#     eth_rsp, geth_rsp = make_same_rpc_calls(eth_rpc, geth_rpc, "eth_getRawTransactionByHash", ["0x5c504ed432cb51138bcf09aa5e8a410dd4a1e204ef84bfed1be16dfba1b22060"])
+#     res, err = same_types(eth_rsp, geth_rsp)
+#     print(eth_rsp)
+#     print(geth_rsp)
+#     assert res, err
```

Comment:
@ramacarlucho @facs95 let's create an issue with all the the RPC endpoints that are missing